src/utils/hints/stateChannelsHints.js

```diff
@@ -12,4 +12,30 @@ export const stateChannelsHints = {
   initiator: 'The account that initiated the creation of the state channel and covered the fees for its creation.',
   responder: 'The participating account that joined the state channel.',
   lastKnownRound: 'Participants can for example protect themselves from being cheated on by providing a snapshot from the latest off-chain round to the chain.',
+  slashedRound: 'The round in which the slash was triggered.',
+  slashedBy: 'The account that slashed (disputed) malicious ChannelCloseSoloTx or ChannelForceProgressTx.',
+  initiatorCreateAmount: 'Unsigned amount of coins the initiator commits to the channel.',
+  initiatorCloseAmount: 'Final balance for the initiator.',
+  initiatorSettleAmount: 'Unsigned amount of coins the initiator gets from the channel.',
+  responderCreateAmount: 'Unsigned amount of coins the responder commits to the channel.',
+  responderCloseAmount: 'Final balance for the responder.',
+  responderSettleAmount: 'Unsigned amount of coins the responder gets from the channel.',
+  settledBy: 'Participant of the channel that posts the settling transaction.',
+  snapshotRound: 'Channel\'s next round.',
```

Comment:
I am not sure if this is a bit misleading and it could be misleading, maybe try:
```suggestion
  snapshotRound: 'Channel snapshot\'s round.',
```